yml/imsim.yaml

```diff
@@ -8022,8 +8022,27 @@ tables:
   - name: ext_trailedSources_Naive_flag_edge
     "@id": "#DiaSource.ext_trailedSources_Naive_flag_edge"
     datatype: boolean
+    description: This flag is set if a trailed source contains edge pixels.
+    fits:tunit:
+    mysql:datatype: BOOLEAN
+  - name: ext_trailedSources_Naive_flag_nan
+    "@id": "#DiaSource.ext_trailedSources_Naive_flag_nan"
+    datatype: boolean
+    description: This flag is set if a trailed source end points contains a nan.
+    fits:tunit:
+    mysql:datatype: BOOLEAN
```

Comment:
Please remove all of these as they are redundant with `datatype: boolean` and will be flagged in the validation as errors in the near future.